server/main.server.js

```diff
@@ -27,12 +27,15 @@ handleDemoUserAccount = function () {
         let demoUser = Meteor.users.findOne({$and: [{username: 'demo'}, {isDemoUser: true}]});
         if (!demoUser) {    // we don't have a demo user, but settings demand one
             Accounts.createUser({username: 'demo', password: 'demo', email: '', profile: {name: 'Demo User'}});
-            Meteor.users.update({'username': 'demo'}, {$set: {isDemoUser: true, isInactive: false}});
+            Meteor.users.update({'username': 'demo'}, {$set: {isDemoUser: true, isInactive: false, 'emails[0].verified': true}});
             console.log('*** ATTENTION ***\n    Created demo/demo user account once on startup');
         } else {    // we already have one, let's ensure he is not switched Inactive
             if (demoUser.isInactive) {
                 Meteor.users.update({'username': 'demo'}, {$set: {isInactive: false}});
             }
+            if (!demoUser.emails[0].verified) {
```

Comment:
This line crashes.
Steps to reproducte:
1. in settings.json:    `createDemoAccount": true` 
1. Perform `meteor reset`
1. Relaunch app (Demo user will be created)
1. Stop app (Ctrl+C)
1. Relaunch app - this line crashes the app, as there is no field `demoUser.emails[0]`

The bug is root caused by this line:
`Meteor.users.update({'username': 'demo'}, {$set: {isDemoUser: true, isInactive: false, 'emails[0].verified': true}});` 

(Hint: compare to your "schema migration #14")